src/scss/includes/direction.scss

```diff
@@ -194,10 +194,30 @@
 
 .itinerary_roadmap_item-block {
   padding: 10px 0;
+
+  &:hover {
+    background-color: $grey-bright;
+    cursor: pointer;
+    position: relative;
+
+    &:before {
+      content: '';
+      position: absolute;
+      top: 0;
+      left: 0;
+      width: 4px;
+      height: 100%;
```

Comment:
The gradient doesn't seem right. In the design, it has 12px vertical margins and rounder corners only on the right. The gradient direction also seems to be top->bottom, contrary to the current `$active-gradient` which is left->right.  
![Capture d’écran de 2020-10-20 15-49-14](https://user-images.githubusercontent.com/243653/96595321-d419b080-12eb-11eb-9687-c817c8314fe6.png)

Reply:
thanks, should be fixed now